Stripe/STPImageLibrary.m

```diff
@@ -52,107 +52,105 @@ + (UIImage *)unknownCardCardImage {
 }
 
 + (UIImage *)brandImageForCardBrand:(STPCardBrand)brand {
```

Comment:
I sorta think we should expose these icons publicly (a la `templatedBrandImageForCardBrand`); they'll be useful to some people.